_studio/mfx_lib/encode_hw/hevc/agnostic/base/hevcehw_base_data.h

```diff
@@ -968,6 +968,16 @@ namespace Base
         }
     };
 
+    /*SliceStructure indicates the restrictions set on the slice structure.*/
+    enum SliceStructure
+    {
+        ONESLICE = 0, // Once slice for the whole frame
+        ROW2ROW = 1, // Slices are power of 2 number of rows, all slices the same
+        ROWSLICE = 2, // Slices are any number of rows, all slices the same
+        ARBITRARY_ROW_SLICE = 3, // Slices are any number of rows, slices can be different
+        ARBITRARY_MB_SLICE = 4, // Slices are any number of MBs, slices can be different
+    };
```

Comment:
@yskuridi , can we keep that awesome comments from _studio/mfx_lib/encode_hw/hevc/agnostic/base/hevcehw_base_legacy_defaults.cpp this in enum?